tests/functional/monitoring/prometheus/metrics/test_ocs_exporter_go_ceph_migration.py

```
@@ -0,0 +1,413 @@
+    logger.info("Detected go-ceph library usage in exporter logs")
+
+
+@runs_on_provider
+@pytest.mark.polarion_id("OCS-6004")
+@blue_squad
+@tier1
+@skipif_external_mode
+@skipif_mcg_only
+@skipif_ms_consumer
+def test_rbd_image_watcher_count_via_go_ceph(
+    exporter_pod, ceph_toolbox_pod, metric_families
```

Ruff (0.15.21)

[warning] Unused function argument: `exporter_pod`

(ARG001)